docs/example/searchbarDocs.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <doc-section
+    id="searchbar"
+    name="searchbar">
```

Comment:
I don't find this ideal 😕, so I think we will need to fix eslint.

Can you help us modify `.eslintrc.js` on your branch to add in the following lines shown below as a separate commit. Then you can put all the attributes on one line instead of having to split to one attribute each line (e.g. `<doc-section id="searchbar" name="searchbar">`)

```diff
diff --git a/.eslintrc.js b/.eslintrc.js
index b74d1ab1..305ef01b 100644
--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -21,5 +21,15 @@ module.exports = {
     'operator-linebreak': ['error', 'before'],
     'quote-props': 'off',
     'vue/order-in-components': 'off',
+    'vue/max-attributes-per-line': [
+      {
+        'singleline': 5,
+        'multiline':
+        {
+          'max': 5,
+          'allowFirstLine': false,
+        },
+      },
+    ],
   },
 };
```

Reply:
id should be `searchbarDocs`, name should be `Searchbar`.